sys/include/net/aodvv2/rfc5444.h

```diff
@@ -105,34 +96,29 @@ typedef enum {
  * @brief   Data about an OrigNode or TargNode.
  */
 typedef struct {
-    ipv6_addr_t addr; /**< IP address of the node */
-    uint8_t metric; /**< Metric value */
-    aodvv2_seqnum_t seqnum; /**< Sequence Number */
+    ipv6_addr_t addr;             /**< IPv6 address of the node */
+    uint8_t pfx_len;              /**< IPv6 address length */
+    uint8_t metric;               /**< Metric value */
+    aodvv2_seqnum_t seqnum;       /**< Sequence Number */
 } node_data_t;
 
 /**
  * @brief   All data contained in a RREQ or RREP.
  */
 typedef struct {
-    uint8_t hoplimit; /**< Hop limit */
-    ipv6_addr_t sender; /**< IP address of the neighboring router which sent the
-                             RREQ/RREP*/
+    uint8_t hoplimit;             /**< Hop limit */
```

Comment:
may be hop_limit ?

Reply:
Looks good `uint8_t hop_limit;`

Reply:
Solved, renamed to `msg_hop_limit` to match lastest draft variable name.